package.json

```diff
@@ -16,7 +16,9 @@
   "author": "Zhen Wang <z@zmxv.com> (http://blog.zmxv.com)",
   "license": "MIT",
   "peerDependencies": {
-    "react-native": ">=0.8.0"
+    "react": ">=16.8.6",
+    "react-native": ">=0.60.0-rc.2",
```

Comment:
does this need to be 64 too?

Reply:
It might work for previous versions.
There's also the question of other platforms as well, which might support lower versions.
This causes warnings at most.

Reply:
thanks for confirming